deploy/docker-compose.yml

```
@@ -41,13 +41,14 @@ services:
       - 42114:42114/tcp
 
   db:
-    build:
-      context: db
+    image: postgres:10-alpine
     restart: unless-stopped
     environment:
       POSTGRES_DB: tinkerbell
       POSTGRES_PASSWORD: tinkerbell
       POSTGRES_USER: tinkerbell
+    volumes:
+      - ./db/tinkerbell-init.sql:/etc/docker-entrypoint-initdb.d/:ro
```

I think this should be
```suggestion
      - ./db/tinkerbell-init.sql:/docker-entrypoint-initdb.d/:ro
```

---

Sorry, actually this:

```suggestion
      - ./db/tinkerbell-init.sql:/docker-entrypoint-initdb.d/tinkerbell-init.sql:ro
```

---

right you are, I thought it docker-compose would to the "obvious" thing.

---

fixed in latest push